imap_processing/codice/codice_l1a.py

```diff
@@ -149,17 +149,16 @@ def define_coordinates(self) -> None:
         """
         self.coords = {}
 
-        for name in self.config["coords"]:
+        coord_names = ["epoch", *list(self.config["dims"].keys())]
```

Comment:
Since `self.config["dims"]` basically has the same values as `self.config["coords"]`, I was able to consolidate here and just use the `dims` list to define the coordinates.